run.py

```diff
@@ -0,0 +1,11 @@
+#!flask/bin/python
+import sys
+from app import app
+
+
+try:
+    host = sys.argv[1]
+except:
+    host = '10.11.12.170'
```

Comment:
i think a saner default is either localhost or interface-wildcard. use the latter, if you want this for yourself.